convert2rhel/actions/system_checks/package_updates.py

```diff
@@ -73,18 +74,36 @@ def run(self):
             # exception is based on the `pkgmanager` module and the message
             # sent to the output can differ depending if the code is running in
             # RHEL7 (yum) or RHEL8 (dnf).
+            package_up_to_date_check_skip = os.environ.get("CONVERT2RHEL_PACKAGE_UP_TO_DATE_CHECK_SKIP", None)
             package_up_to_date_error_message = (
                 "There was an error while checking whether the installed packages are up-to-date. Having an updated system is"
                 " an important prerequisite for a successful conversion. Consider verifyng the system is up to date manually"
                 " before proceeding with the conversion. %s" % str(e)
             )
+            if not package_up_to_date_check_skip:
+                logger.warning(package_up_to_date_error_message)
+                self.set_result(
+                    level="OVERRIDABLE",
+                    id="PACKAGE_UP_TO_DATE_CHECK_FAIL",
+                    title="Package up to date check fail",
+                    description="Please refer to the diagnosis for further information",
+                    diagnosis=package_up_to_date_error_message,
+                )
+                return
+            skip_package_up_to_date_check_message = (
+                "Detected 'CONVERT2RHEL_PACKAGE_UP_TO_DATE_CHECK_SKIP' environment variable, we will skip "
```

Comment:
Did we want to use backticks here instead of single-quote? @r0x0d does this get converted in the UI?